src/states/menus/lobby.gd

```diff
@@ -125,8 +132,10 @@ func on_back_button_press() -> void:
 @rpc("any_peer")
 func on_start_button_press() -> void:
 	if is_multiplayer_authority():
-		Multiplayer.unready_players()
-		start_game.rpc()
+		var remote_id := multiplayer.get_remote_sender_id()
+		if remote_id == 0 or Multiplayer.is_id_leader(remote_id):
+			Multiplayer.unready_players()
```

Comment:
Missed rename of this function to `mark_players_as_unloaded` or smth

Reply:
donezo